azure/src/createAzureClient.ts

```diff
@@ -356,7 +356,7 @@ class AzExtBearerChallengePolicy implements PipelinePolicy {
                 // fetching fails, we don't attempt the challenge again.
                 request.headers.set(this.challengeRetryHeader, '1');
 
-                const token = await this.getTokenForChallenge({ challenge: header, wwwAuthenticate: header, scopes });
+                const token = await this.getTokenForChallenge({ challenge: header, wwwAuthenticate: header, fallbackScopes: scopes, scopes } as unknown as vscode.AuthenticationWwwAuthenticateRequest);
```

Comment:
Operative change here. I did an `as unknown as vscode.AuthenticationWwwAuthenticateRequest` so it wouldn't get mad about me passing `challenge`, which technically no longer exists in the current version of `vscode.AuthenticationWwwAuthenticateRequest`.